Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y openssh-server
+
+# Env Variables to pass username and password.
+ARG USERNAME=sftp_user
+ARG PASSWORD=Testing123
```

Comment:
instead of hardcoded password use public key authentication and mount public key as authorized keys in runtime.